contracts/Unibright.sol

```diff
@@ -2,288 +2,251 @@
- * to run tests before sending real value to this contract.
  */
 contract UBTSplitter is Context, Ownable {
-    using Address for address;
     event PayeeAdded(
         address revenueAddress,
         address stakingAddress,
         uint256 shares,
-        string baseledgervaloper,
-        uint256 state_lastEventNonce,
+        string baseledgerValidatorAddress,
+        uint256 lastEventNonce,
         uint256 timestamp
     );
+
     event PayeeUpdated(
         address revenueAddress,
         address stakingAddress,
         uint256 shares,
-        string baseledgervaloper,
-        uint256 state_lastEventNonce,
+        string baseledgerValidatorAddress,
+        uint256 lastEventNonce,
         uint256 timestamp
     );
-    event WhitelistTokenUpdated(
-        address triggerredBy,
-        address token,
-        bool isWhitelisted
-    );
 
-    event ERC20PaymentReleased(
+    event UbtPaymentReleased(
         IERC20 indexed token,
         address revenueAddress,
         address stakingAddress,
         uint256 amount
     );
 
-    event ERC20Deposit(
+    event UbtDeposited(
         address sender,
         address token,
         uint256 tokenAmount,
-        uint256 state_lastEventNonce,
+        uint256 lastEventNonce,
         string destinationAddress
     );
 
     uint256 public totalShares;
-    uint256 public state_lastEventNonce;
+    uint256 public lastEventNonce;
 
     mapping(address => uint256) public shares;
-    mapping(address => address) public validatorStakingAddress; // revenueAddress => validatorStakingAddress
-    address[] public payees;
+    mapping(address => address) public stakingAddresses;
+    mapping(address => uint256) public ubtReleased;
 
-    mapping(IERC20 => uint256) public erc20TotalReleased;
-    mapping(IERC20 => mapping(address => uint256)) public erc20Released;
-    
-    address public theWhitelistedToken;
+    mapping(address => bool) public payees;
 
-    uint256 public theErc20ToBeReleasedInPeriod;
-    uint256 public theErc20NotReleasedInLastPeriod;
-    uint256 public theErc20CurrentPeriod;
-
-    mapping(uint256 => mapping (address => uint256)) theErc20ReleasedPerRecipientInPeriods;
+    uint256 public ubtTotalReleased;
+    mapping(uint256 => mapping (address => uint256)) public ubtReleasedPerRecipientInPeriods;
+    
+    uint256 public ubtToBeReleasedInPeriod;
+    uint256 public ubtNotReleasedInLastPeriods;
+    uint256 public ubtCurrentPeriod;
 
+    address public whitelistedToken;
 
-    /**
-     * @dev Creates an instance of `UBTSplitter` where each account in `payees` is assigned the number of shares at
-     * the matching position in the `shares` array.
-     *
-     * All addresses in `payees` must be non-zero. Both arrays must have the same non-zero length, and there must be no
-     * duplicates in `payees`.
-     */
     constructor(address token) {
-        theWhitelistedToken = token;
+        whitelistedToken = token;
```

Comment:
not sure if we should call it whitelistedToken anymore?

Reply:
ubtTokenContractAddress might be a better name